.github/actions/check-current-workflow-is-latest/action.yml

```diff
@@ -26,6 +26,6 @@ runs:
       shell: pwsh
       id: check-current-workflow-equals-latest
       run: |
-        $latestWorkflowRunDatabaseId = $(gh run list --branch ${{ github.head_ref }} --limit 1 --json databaseId | ConvertFrom-Json)[0].databaseId
+        $latestWorkflowRunDatabaseId = $(gh run list -R '${{ github.repository }}' -b '${{ github.head_ref }}' -w '${{ github.workflow }}' -L 1 --json databaseId | ConvertFrom-Json)[0].databaseId
```

Comment:
Line is too long: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/43384918/PowerShell+code+conventions#Formatting.

Also, use unabbreviated parameter names so the command is easier to read.

Reply:
The docs I linked clearly state that you should use splatting with arrays in this case like:

```
$dockerRunSwitches = @(
    '--name', 'sql2019'
    '--env', 'ACCEPT_EULA=Y'
    '--env', 'SA_PASSWORD=Password1!'
    '--publish', '1433:1433'
    '--detach', 'mcr.microsoft.com/mssql/server:2019-latest'
)

docker run @dockerRunSwitches
```

So please use that.

Reply:
Well, I missunderstood the docs. Now it makes sense. For some reason I thought that the Hash Table version should be used in this case.